.github/scripts/checkKnipRegression.ts

```diff
@@ -0,0 +1,60 @@
+import * as $ from 'shelljs'
+
+const SECTION_REGEX = /[a-zA-Z ]+\([0-9]+\)/
+const CATEGORY_REGEX = /[a-zA-Z ]+/
+const PARENS_REGEX = /\(|\)| /
+
+function parseKnipOutput(knipOutput: string): Record<string, number> {
```

Comment:
would be cool if this had a test that just included some output you've seen from knip